CHANGELOG.md

```diff
@@ -16,6 +16,12 @@
 
 ---
 
+## v2.18.0
+
+- 👷 Versioned CDN files ([#932](https://github.com/DataDog/browser-sdk/pull/932))
+- 👷 Increase browser cache to 4 hours ([#933](https://github.com/DataDog/browser-sdk/pull/933))
+- 📝 add `actionNameAttribute` initialization parameter ([#923](https://github.com/DataDog/browser-sdk/pull/923))
```

Comment:
did you remove `🐛 ensure that test cookie is correctly deleted after the test (#927)`?

Reply:
My bad, I thought it was only for tests...

Reply:
I'll add it. Are the 3 other points good for you?

Reply:
LGTM!